public/app/features/alerting/unified/components/silences/AmAlertStateTag.tsx

```diff
@@ -12,6 +12,11 @@ const alertStateToState: Record<AlertState, State> = {
 
 interface Props {
   state: AlertState;
+  // @PERCONA
+  silenced?: string;
 }
 
-export const AmAlertStateTag: FC<Props> = ({ state }) => <StateTag state={alertStateToState[state]}>{state}</StateTag>;
+// @PERCONA
+export const AmAlertStateTag: FC<Props> = ({ state, silenced }) => (
+  <StateTag state={alertStateToState[state]}>{state === 'suppressed' ? silenced : state}</StateTag>
```

Comment:
I think it would be good to have a condition also for the silenced string not to be empty.

If the state was `'supressed'` and `silenced` was empty, it would render out an empty body. Not sure if we want to do that for other pages/components that use the AmAlertStateTag (outside of percona integrated alerting).

Reply:
Good catch. I have confirmed with Roma that only this one page needs to be updated for suppressed -> silenced hence added the condition for empty silenced prop.